src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -301,22 +314,39 @@ private static string DecodePropertyName(string name, string language)
             return name;
         }
 
-        private class DeclarationGenerator : SymbolVisitor
+        /// <summary>
+        /// The <see langword="bool"/> generic type argument indicates if we succeeded at writing out the declaration ID
+        /// or not.  Callers should only call into <see cref="SymbolVisitor{TResult}.Visit(ISymbol?)"/> and should check
+        /// <see cref="Failed"/> to see if it failed (in the case of an arbitrary symbol) or that it produced an
```

Comment:
><see cref="Failed"/>  [](http://example.com/codeflow?start=12&length=21)

It looks like the flag is not going to be reset for a subsequent ```Visit``` calls. It might be worth mentioning the fact in this comment,

Reply:
sure!